app/uk/gov/hmrc/soletraderidentificationfrontend/views/details_not_found_page.scala.html

```diff
@@ -31,25 +32,28 @@
 
 @layout(pageConfig, title = title("details-not-found-page.title")) {
 
-    @h1(messages("details-not-found-page.heading"))
-
+    @p{@h1(messages("details-not-found-page.heading"))}
```

Comment:
I would not change this. The h1 function defines the appropriate CSS styling. Further, I don't think the <br> is necessary, the paragraph function again defines the necessary styling.